tests/test_tokenization.py

```diff
@@ -133,6 +133,60 @@ def fetch_from_2(self, query, parameters={}, literal_binds=True):
         query, parameters = self.compileQuery(query, parameters=parameters, literal_binds=literal_binds)
         return self.executor2.execute_prepared_statement(query, parameters)
 
+    def compile_execute_prepare(self, prepared_name, data={}):
+        if len(data) == 0:
+            prepare_query_sql = "execute {}".format(prepared_name)
+        else:
+            values_str = ''
+            for x in data.values():
+                if isinstance(x, int):
```

Comment:
did you try [sa.literal](https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.literal) to delegate escaping and conversion to sqlalchemy?

Reply:
sqlalchemy can compile queries to the string, and name it like `bind values`. plus, such objects convertible to the string via `str(obj)`. And when they have assigned engine or dialect, some of them serialize to the target syntax

Reply:
I guess you tried and it not works)